stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -540,6 +540,17 @@ public type Hub object {
     public string publishUrl;
     private http:Listener hubHttpListener;
 
+    # The initialization method for the Hub. Users of the `ballerina/websub` module must use the
+    # function `websub:startHub()` to initialize a `websub:Hub` object.
+    #
+    # + subscriptionUrl - The URL for subscription changes
+    # + publishUrl - The URL for publishing and topic registration
+    # + hubHttpListener - The `http:Listener` to which the hub service should be attached
+    #
+    # # Deprecated
+    # Users of the `ballerina/websub` module must use the function `websub:startHub()` to initialize a `websub:Hub`
+    # object, instead of directly calling the initializer method.
```

Comment:
```suggestion
    # object instead of directly calling the initializer method.
```

Reply:
@MaryamZi - This has been missed.

Reply:
No, I've done this change. Please check https://github.com/ballerina-platform/ballerina-lang/pull/22187/commits/c666b802279a6d5a241c911d1eb321ee78771627#diff-93219952a83119ddf33cd65933550586R552

Reply:
+1